compiler/model/utils.ts

```diff
@@ -292,7 +292,7 @@ export function modelType (node: Node): model.ValueOf {
         default: {
           const generics = node.getTypeArguments().map(node => modelType(node))
           const identifier = node.getTypeName()
-          assert(node, Node.isIdentifier(identifier), 'Not an identifier')
+          assert(node, Node.isIdentifier(identifier), 'Not an codegen_name')
```

Comment:
"Not a valid identifier for codegen_name" would me more explicit. Compiler messages are sometimes so succinct that they're hard to understand 😉